packages/reactive_ble_platform_interface/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Main releases
 
+## 4.0.1
+
+ * Add support for iOS 15
```

Comment:
The first character at this line is space. It's not needed for a top-level list item in Markdown. Also, compare with older items.